Form-Controls/styles.css

```diff
@@ -0,0 +1,232 @@
+    border-radius: 4px;
+    display: block;
+    width: 100%;
+    padding: 10px;
+    font-size: 14px;
+}
+
+.form-control input:focus{
+    outline: 0;
+    border-color: #777;
+}
+
+/* ------------------- validation email ----------------------- */
+
+input:invalid:focus {
+    border-color:red;
+    }
+    
+input:valid:focus {
+    border-color: green;
+    }
+
+
+/* ------------------- radio button size ----------------------- */
+
+.form-size,
+.dropdown {
+    border: 2px solid #f0f0f0;
+    border-radius: 9px;
+    display: block;
+    width: 100%;
+    padding: 15px;
+    font-size: 14px;
+}
+
+#red {
+    accent-color: red;
+}
+
+#yellow {
+    accent-color: yellow;
+}
+
+#Blue {
+    accent-color: blue;
+}
+
+/* ------------------- Select dropdown ----------------------- */
+
+select {
+    padding: .5rem;
+    font-size: 12px;
+    color: #ffffff;
+    background-color: #3498db;
+    border: none;
+    cursor: pointer;
+    width: 15%;
+}
+
+option {
+    background-color: #f9f9f9;
+    color: black;
+    padding: 10px 10px;
+}
+
+.select:focus {
+    outline: none;
+    background-color: #3e8e41;
+}
+
+select:hover {
+    background-color: rgba(8, 249, 217, 0.357);
+    color: black;
+    font-weight: bold !important;
+}
+
+/* ------------------- form-control calendar ----------------------- */
+
+.form-delivery{
+    border: 2px solid #f0f0f0;
+    border-radius: 4px;
+    display: block;
+    width: 100%;
+    padding: 10px;
+    font-size: 14px;
+}
+
+.form-delivery input {
+    margin-top: 1rem;
+    background-color: rgba(8, 249, 217, 0.357);
+    color: rgb(24, 11, 64);
+    font-size: larger;
+}
+
+/* ------------------- submit btn ----------------------- */
+
+.form-btn {
+    background-color: #3498db;
+    border: none; 
+    color: white; 
+    padding: 10px 20px; 
+    text-align: center; 
+    text-decoration: none; 
+    display: block; 
+    font-size: 16px; 
+    margin-top: 20px;
+    cursor: pointer; 
+    border-radius: 5px;
+    transition: background-color 0.3s;
+    box-shadow: 0 2px 10px;
+    width: 100%;
+}
+
+button:hover {
+    background-color: hsl(173, 100%, 50%);
+    color: black;
+}
+
+
+/* ------------------- footer -------------------------------------- */
+
+
+footer {
+    text-align: center;
+    margin: 10rem 4rem;
+}
+
+
+/* ------------------- Media query ----------------------- */
+
+
+@media (max-width: 768px) {
+    .container {
+        width: 90%;
+        padding: 10px;
+    }
+
+    .form {
+        padding: 20px;
+    }
+
+    select {
+        width: 20%;
+    }
+
+    .form-size label,
+    .dropdown label {
+        margin-right: 5px;
+        font-size: 14px;
+    }
+}
+
+@media (max-width: 480px) {
+    .container {
+        width: 85%;
+        padding: 1rem;
+    }
+
+    .form {
+        padding: 10px;
+    }
+
+    select {
+        font-size: 14px;
+        width: 40%;
+    }
+
+    .form button {
+        font-size: 14px;
+        padding: 12px;
```

Comment:
Hi @Grajales-K ,
Same as previous comment.

Reply:
Thanks, @mrvicthor. I updated all px to rem but left the border radius and shadow in px.